hack/deployer/runner/settings.go

```diff
@@ -64,6 +64,7 @@ type OcpSettings struct {
 	Region        string `yaml:"region"`
 	AdminUsername string `yaml:"adminUsername"`
 	WorkDir       string `yaml:"workDir"`
+	PullSecret    string `yaml:"PullSecret"`
```

Comment:
```suggestion
	PullSecret    string `yaml:"pullSecret"`
```

Reply:
argh, forgot to change the capitalization